src/main/resources/reference.conf

```diff
@@ -500,7 +499,7 @@ target-validation {
 disease {
   efo-ontology {
     format = "json"
-    path = ${common.input}"/ontology-inputs/ontology-efo-"${efo_version}".jsonl"
+    path = ${common.input}"/ontology-inputs/ontology-efo.jsonl"
```

Comment:
I don't think it's a problem not having the EFO version here, but do we know from the PIS logs which version was downloaded?

Reply:
It's in PIS configuration, and the URL is logged

Reply:
```
2022-10-24 15:12:43,960 yapsy_loaded_plugin_Disease_0 INFO - [STEP] BEGIN, Disease
2022-10-24 15:12:43,960 modules.common.DownloadResource DEBUG - Start to download
        https://github.com/EBISPOT/efo/releases/download/v3.45.0/efo_otar_slim.owl 
2022-10-24 15:12:43,961 modules.common.DownloadResource INFO - [DOWNLOAD] BEGIN: 'https://github.com/EBISPOT/efo/releases/download/v3.45.0/efo_otar_slim.owl' -> '/srv/output/staging/efo_otar_slim.owl'
2022-10-24 15:12:52,505 modules.common.DownloadResource INFO - [DOWNLOAD] END: 'https://github.com/EBISPOT/efo/releases/download/v3.45.0/efo_otar_slim.owl' -> '/srv/output/staging/efo_otar_slim.owl'
2022-10-24 15:12:52,505 yapsy_loaded_plugin_Disease_0 DEBUG - OWL to JSON output path '/srv/output/staging/ontology-inputs'
2022-10-24 15:12:52,505 modules.common.Riot DEBUG - RIOT OWL file '/srv/output/staging/efo_otar_slim.owl' to JSON, output folder '/srv/output/staging/ontology-inputs', destination file name 'efo_otar_slim.json', JQ filter '.["@graph"][] | select (.["@type"] == ("Class","Restriction") and ."@id" != null)|.subClassOf=([.subClassOf]|flatten|del(.[]|nulls))|.hasExactSynonym=([.hasExactSynonym]|flatten|del(.[]|nulls))|.hasDbXref=([.hasDbXref]|flatten|del(.[]|nulls))|.inSubset=([.inSubset]|flatten|del(.[]|nulls))|.hasAlternativeId=([.hasAlternativeId]|flatten|del(.[]|nulls))|@json'
```